callouts/python/extproc/example/jwt_auth/service_callout_example.py

```diff
@@ -53,7 +53,16 @@ class CalloutServerExample(callout_server.CalloutServer):
   def on_request_headers(
       self, headers: service_pb2.HttpHeaders,
       context: ServicerContext):
-    """Custom processor on request headers."""
+    """Custom processor on request headers.
```

Comment:
I can see that the docstrings from the base class have been repeated, would removing the inherited string altogether allow for sphinx to use the correct docstrings?

Reply:
According to their documentation it was supposed to inherit the docs when leaving it blank but I tried with different ways and it was not pulling it from the original, I tried another approach, I linked the original methods and you can navigate through them by just clicking, you can check it here:
https://pweiber.github.io/service-extensions-samples/python/_build/extproc/jwt_auth/service_callout_example.html

let me know what you think, if that's fine we can do it for the other ones as well.

If the layout is still messed up, refreshing the cache should fix it.

Reply:
If this is referring to the green [source] button that is fine as an alternative yes.